spec/helpers/application_helper_spec.rb

```diff
@@ -94,4 +94,27 @@
       end
     end
   end
+
+  describe 'page title helper' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.